<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitComponentTest</class>
 <widget class="QWidget" name="UnitComponentTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UnitComponentTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>表格</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
         <item>
          <widget class="CusTableView" name="widget" native="true"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>网格</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,1">
       <item>
        <widget class="FlowLayoutArea" name="FlowFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>翻页按钮</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="TablePageButton" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>局部翻页</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="9,1">
       <item>
        <widget class="TurnPageGridView" name="turnPageViewWidget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="turnPage_btn1">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="turnPageBtn_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>左右调整</string>
      </attribute>
      <widget class="CusAdjustBox" name="frame_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>日历</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="CusCalendarBox" name="widget_2" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>测试</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 127);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>页</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="CusLineChartWidget" name="widget_4" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CusTableView</class>
   <extends>QWidget</extends>
   <header location="global">CusTableView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TurnPageGridView</class>
   <extends>QWidget</extends>
   <header location="global">TurnPageGridView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FlowLayoutArea</class>
   <extends>QFrame</extends>
   <header location="global">FlowLayoutArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TablePageButton</class>
   <extends>QFrame</extends>
   <header location="global">TablePageButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CusAdjustBox</class>
   <extends>QFrame</extends>
   <header location="global">CusAdjustBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CusCalendarBox</class>
   <extends>QWidget</extends>
   <header location="global">CusCalendarBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CusLineChartWidget</class>
   <extends>QWidget</extends>
   <header location="global">CusLineChartWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
